examples/grpc-samples/deploy-helpers.sh

```diff
@@ -0,0 +1,42 @@
+export PROJECT_PATH="$( dirname $( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd ) )"
```

Comment:
@sourishkrout are you sure you want to keep this file?

Reply:
Yes, otherwise https://github.com/stateful/runme/blob/expression-in-double-quotes/examples/grpc-samples/services.md#complex-script won't run.